google/cloud/spanner_v1/pool.py

```diff
@@ -423,16 +425,18 @@ def ping(self):
         """
         while True:
             try:
-                ping_after, session = self._sessions.get(block=False)
+                last_used, session = self._sessions.get(block=False)
             except queue.Empty:  # all sessions in use
                 break
-            if ping_after > _NOW():  # oldest session is fresh
+            if last_used + self._delta > _NOW():  # oldest session is fresh
```

Comment:
Does the old code have a bug here? In the old code, it doesn't add `self._delta`.

Reply:
In the original, the stored value was _NOW() + self._delta which is why it was called ping after. I changed the stored value to _NOW() so I could tell when a session had been unused for an hour.